subsys/suit/stream/stream_sinks/src/suit_ram_sink.c

```diff
@@ -10,6 +10,13 @@
 #include <string.h>
 #include <suit_ram_sink.h>
 #include <suit_memory_layout.h>
+#include <zephyr/cache.h>
+
+#ifdef CONFIG_DCACHE_LINE_SIZE
+#define CACHE_ALIGNMENT CONFIG_DCACHE_LINE_SIZE
```

Comment:
Why not use `sys_cache_data_line_size_get()`? That way you also don't assume a hardcoded 4 in case of a misconfiguration